content/en/docs/operator.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Operators
+menu:
+  main:
+    weight: 10
+---
+
+## What is an Operator?
+
+An Operator is a method of packaging, deploying and managing a Kubernetes-native application. A Kubernetes-native application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. An Operator is a piece of software that runs in a Pod inside the Kubernetes cluster and it interacts with the Kubernetes API. It has a reconciler loop that runs continuously, which monitors Objects (Pods, Services, ConfigMaps, or PersistentVolumes) for a desired state given by a Custom Resource Definition (CRD) and replicates that state in the cluster.
+
+Conceptually, Operators take human operational knowledge and encode it into software that is more easily shared with consumers.
+
+Operators are pieces of software that ease the operational complexity of running another piece of software. They act like an extension of the software vendor’s engineering team, watching over a Kubernetes environment (such as OpenShift Container Platform) and using its current state to make decisions in real time. Advanced Operators are designed to handle upgrades seamlessly, react to failures automatically, and not take shortcuts, like skipping a software backup process to save time.
+
+More technically, Operators are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an app that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` or `oc` tooling. To be able to make the most of Kubernetes, you require a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
```

Comment:
I am not sure if I agree with all that is defined here. And then, IMO we should leverage more in K8S docs instead of trying to create our own definitions in this case. Following my suggestion.

```suggestion
Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extendig the Kubernetes API with CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and managed them by looking for the desired state. For a better understanding see [Operator Pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
```